host/class/uac/usb_host_uac/test_app/sdkconfig.defaults

```diff
@@ -0,0 +1,8 @@
+# This file was generated using idf.py save-defconfig. It can be edited manually.
+# Espressif IoT Development Framework (ESP-IDF) Project Minimal Configuration
+#
+CONFIG_ESPTOOLPY_FLASHSIZE_4MB=y
+CONFIG_PARTITION_TABLE_CUSTOM=y
+CONFIG_FREERTOS_HZ=1000
+CONFIG_USB_HOST_CONTROL_TRANSFER_MAX_SIZE=2048
+CONFIG_USB_HOST_HW_BUFFER_BIAS_PERIODIC_OUT=y
```

Comment:
In addition to providing three FIFO allocation logic, ESP-IDF need more flexible support for user configuration of FIFO sizes, so that we can provide more flexible device support.

I have a USB Headset with IN MPS 192, OUT MPS 384, there is no FIFO configuration that allows two interfaces to be used simultaneously.